app/components/widgets/inputs/AlertLayoutInput.vue.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, Prop } from 'vue-property-decorator';
+import { Inject } from 'util/injector';
+import { IListMetadata } from '../../shared/inputs';
+import { BaseInput } from 'components/shared/inputs/BaseInput';
+import ImagePickerInput from '../../shared/inputs/ImagePickerInput.vue';
+import { CustomizationService } from 'services/customization';
+
+@Component({
+  components: { ImagePickerInput }
+})
+export default class ImageLayoutInput extends BaseInput<string, IListMetadata<string>> {
```

Comment:
This class name doesn't match the file name.  This results in very confusing error messages from Vue when this component raises an exception.